wire-runtime/src/test/proto-java/com/squareup/wire/protos/person/Person.java

```diff
@@ -219,6 +220,22 @@ public static PhoneType fromValue(int value) {
     public int getValue() {
       return value;
     }
+
+    private static final class ProtoAdapter_PhoneType extends EnumAdapter<PhoneType> {
```

Comment:
Do we use this naming convention elsewhere? If not, I would prefer something like PhoneTypeEnumAdapter that's more human-friendly.